import collections

Error = collections.namedtuple("ConfigError", ["source", "message", "entry"])
